src/gui_gtk_f.c

```diff
@@ -258,14 +257,15 @@ gui_gtk_form_class_init(GtkFormClass *klass)
     container_class->forall = form_forall;
 }
 
-    static void
+static void
 gui_gtk_form_init(GtkForm *form
-#if !GTK_CHECK_VERSION(3,0,0)
-	, void *g_class UNUSED
+#if !GTK_CHECK_VERSION(3, 0, 0)
+                  ,
+                  void *g_class UNUSED
```

Comment:
this looks harder to read, no? I suppose that is not easily to fix however?

Reply:
Didn't catch that! Yes, I agree looks really hard to read. Will fix probably tonight (I'm in the `Europe/Rome` timezone).